batch_conversion.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/env python
+
+"""URDF files to Webots PROTO converter."""
```

Comment:
It would be nice to give a small explanation here about how to use this script.

Reply:
noted, I'll work on that

Reply:
What logic are you following when using """ instead of #?

Reply:
This one: https://www.python.org/dev/peps/pep-0257/